palantir-java-format/src/main/java/com/palantir/javaformat/doc/Level.java

```diff
@@ -40,7 +40,7 @@
      * How many branches we are allowed to take (i.e how many times we can consider breaking vs not breaking the current
      * level) before we stop branching and always break, which is the google-java-format default behaviour.
      */
-    private static final int MAX_BRANCHING_COEFFICIENT = 7;
+    private static final int MAX_BRANCHING_COEFFICIENT = 20;
```

Comment:
@dansanduleac explained in person this can be dialled up (and still be performant) because the formatter was not previously counting branching accurately (so was actually doing lots more work than a max of 7 would have allowed!)